mixer/adapter/bypass/bypass.go

```diff
@@ -159,12 +160,17 @@ func (b *builder) Build(context context.Context, env adapter.Env) (adapter.Handl
 }
 
 func (b *builder) ensureConn() error {
+	const timeOut = time.Minute
+	const delay = time.Millisecond * 10
+
 	if b.conn == nil {
-		conn, err := grpc.Dial(b.params.BackendAddress, grpc.WithInsecure())
-		if err != nil {
-			return err
+		var err error
+		for start := time.Now(); time.Now().Before(start.Add(timeOut)); {
```

Comment:
We have a few retry libraries lying around ... maybe use one of those?

Also, there's a pretty lightweight lib for this sort of thing: https://github.com/cenkalti/backoff/blob/master/retry.go